client/js/sorting.js

```diff
@@ -17,18 +17,15 @@ module.exports = function() {
 		forcePlaceholderSize: true,
 		tolerance: "pointer", // Use the pointer to figure out where the network is in the list
 
-		update: function() {
+		update() {
+			const type = "networks";
```

Comment:
This PR looks good now, except for this (and line 44). It feels somewhat unnatural to do it like this.

Reply:
Ok, reverted both.